aiida/cmdline/params/options/__init__.py

```diff
@@ -33,6 +33,18 @@
     'WITH_ELEMENTS_EXCLUSIVE'
 )
 
+TRAVERSAL_RULE_HELP_STRING = {
+    'call_calc_backward': 'CALL links to calculations backwards',
```

Comment:
There are 3 rules missing. I would just add them so that if at some point they are made toggleable this won't crash

Reply:
will add!